docker/integration_tests/dockerScripts/AppsContainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mono:5.12
+FROM microsoft/dotnet:sdk AS build-env
```

Comment:
No need to define a name such as build-env.
It is not a multi stage dockerfile.

Reply:
i do it